### DISCUSSION TITLE
How can I listen to the WIKI loaded action?

### DISCUSSION THREAD

**Discussion by scienceasdf** [requarks/wiki#2838] [Help / Questions] [answered]
When I want to write scripts on the page, I have no way to listen to the WIKI loading. As a result, I have to delay some function a few seconds after document is ready, as the WIKI DOM elements are not ready. Is there a way to get the WIKI DOM element are all ready?

**Comment by NGPixel** [comment, marked as answer]
Assuming you're running code on/after domready:

```js
window.boot.register('vue', () => {
  // YOUR CODE HERE
})
```

**Reply by DictXiong** [reply]
Not working for me. In my case, `window.boot.register` is triggered even before `document.ready`. At that time, the page is still not loaded completely. 

Using wiki.js 2.5.283 in docker on ubuntu 20.04

![image](https://user-images.githubusercontent.com/41772157/173176423-8574e8ea-603f-42c0-ac56-0585c685b517.png)
